scripts/Muon/thread_model.py

```diff
@@ -17,8 +20,13 @@ def __del__(self):
         self.wait()
 
     def run(self):
-        self.model.execute()
-        self.model.output()
+        try:
+            self.model.execute()
+            self.model.output()
+        except:
```

Comment:
I think it is considered good practice to specify the type of exception you want to catch here. In this case it appears to be KeyboardInterrupt that you want to catch.